exo-go/src/aws_helpers/index.go

```diff
@@ -20,6 +26,23 @@ func hasBucket(s3client *s3.S3, bucketName string) (bool, error) {
 	return false, err
 }
 
+// create s3 bucket if it doesn't already exist
+func createBucket(s3client *s3.S3, bucketName string) error {
+	hasBucket, err := hasBucket(s3client, bucketName)
+	if err != nil {
+		return err
+	}
+
+	if !hasBucket {
+		_, err = s3client.CreateBucket(&s3.CreateBucketInput{Bucket: &bucketName})
+		if err != nil {
+			return err
+		}
+	}
+
+	return err
```

Comment:
Also thoughts on removing all the empty line in functions? I think its better to keep them with fewer lines and that should hopefully make us focus on extracting out helper methods as needed as you don't get the empty lines to visually separate the different parts. Same for all the files in this PR